src/Umbraco.Web/PropertyEditors/ValueListConfigurationEditor.cs

```diff
@@ -68,30 +68,41 @@ public override ValueListConfiguration FromConfigurationEditor(IDictionary<strin
         {
             var output = new ValueListConfiguration();
 
-            if (!editorValues.TryGetValue("items", out var jjj) || !(jjj is JArray jItems))
-                return output; // oops
+            if (editorValues.ContainsKey("items") == false)
+                return output;
+
+            var jsonObject = (JObject) editorValues["items"];
```

Comment:
this is going to throw if the value is not JObject

Reply:
Its only going to throw if we get something else than expected. What is expected in this case however is different, from whom the caller is.